operator-framework/src/test/java/io/javaoperatorsdk/operator/baseapi/informerremotecluster/InformerRemoteClusterReconciler.java

```diff
@@ -53,7 +53,8 @@ public List<EventSource<?, InformerRemoteClusterCustomResource>> prepareEventSou
         .from(ConfigMap.class, InformerRemoteClusterCustomResource.class)
         // owner references do not work cross cluster, using
         // annotations here to reference primary resource
-        .withSecondaryToPrimaryMapper(Mappers.fromDefaultAnnotations())
+        .withSecondaryToPrimaryMapper(
+            Mappers.fromDefaultAnnotations(InformerRemoteClusterCustomResource.class))
```

Comment:
Maybe we could eliminate the part where we pass the class of primary resource. Maybe a setter on `SecondaryToPrimaryMapper` for the primary class?  
@metacosm